sccp/figures/figureThomsonDrop.py

```diff
@@ -1,40 +1,228 @@
+"""
+Describes the model's reaction to the removal of specific pieces of the
+dataset. It can clearly be observed for all cell types that had an identifying component that the
+corresponding weight for the condition removed significantly dropped.
+"""
+
 from .common import getSetup
 from ..imports import import_thomson
 from ..factorization import pf2
-from .commonFuncs.plotFactors import (
-    plotConditionsFactors,
-    plotCellState,
-    plotGeneFactors,
-)
-from .figureThomson1 import groupDrugs
+from ..imports import import_thomson
+from .commonFuncs.plotFactors import reorder_table
+import pandas as pd
+import numpy as np
+import seaborn as sns
+from ..gating import marker_genes_1, marker_genes_2
+from anndata import AnnData
+from matplotlib.axes import Axes
+from scipy.stats import linregress
 
 
 def makeFigure():
     rank = 20
     data = import_thomson()
-
-    sampled_data = data[
-        (data.obs["Cell Type"] != "T Cells") | (data.obs["Condition"] != "CTRL4")
+    
+    ax, f = getSetup((10, 10 * 9), (1 * 9, 2))
```

Comment:
@Nathaniel-github I would change it be much smaller. The left parenth. are the size of the whole figure and right is the how many figures you want